onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4057,35 +4022,46 @@ def aten_native_group_norm_backward(
 @torch_op("aten::native_layer_norm", trace_only=True)
 def aten_native_layer_norm(
     input: TReal,
-    normalized_shape: INT64,
+    normalized_shape: Sequence[int],
```

Comment:
Should this be Sequence[INT64]? I am confused.

Reply:
A couple of comments:
(i) `Sequence[INT64]` is very unlikely in most ops. `Sequence[int]` (for something to be represented as an attribute in ONNX) and `INT64` (for something to be represented as an input in ONNX) are much more likely. 
(ii) `INT64` itself is better represented as `INT64[...]` (for unknown rank) or as appropriate if rank/shape is known. However,  this has not been done much so far in any of the library, so this may be something we can get to at some point.

Reply:
Ok, I am reading the other issue (https://github.com/microsoft/onnx-script/pull/517 ) only now, so, please ignore my comment above. Will rethink this.

Reply:
Reverted to INT64

Reply:
INT64[any thing] is not valid type for now so I am avoiding until we fix typing